app/models/mongoidable/ability.rb

```diff
@@ -16,15 +16,15 @@ class Ability
     # Extra arguments as defined by cancancan.
     field :extra, type: Array
 
-    validates_presence_of :action
+    validates :action, presence: true
     validate :subject do |object|
-      raise ArgumentError, ":subject cannot be blank" if object.subject.nil?
+      raise ArgumentError, _(":subject cannot be blank") if object.subject.nil?
```

Comment:
not familiar with `_()`.  What does this do?

Reply:
+1